modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/RemoteRegistryService.java

```diff
@@ -160,4 +162,29 @@ public abstract void copyNonExistingResources(String sourceServerUrl, String sou
 	                                              String destServerUrl, String appId, String username)
 			throws AppFactoryException;
 
+	/**
+	 * Create runtime properties on Kubernetes for a application
```

Comment:
A javadoc comment should end with the period. Else this will throw error in a windows build. Fix these type of javadoc issues on all classes.